### DISCUSSION TITLE
All my editor mode feedback (with priority numbering 1-5)

### DISCUSSION THREAD

**Discussion by AliveJy** [Mojang/minecraft-editor#62] [Feedback]
After trying it out a lot, I came with a list of things that I think should be in editor mode (with priority numbering)

~2 Spherical Brushes: Would make builds a lot smoother than the current square-only brushes.

~4 Block item IDs inserting: Some blocks are inaccessible in editor mode due to the lack of block item ID inserting.

~3 Replace block with block in a selected region: When selecting a region, there should be a option to replace (block) with (block) in a selected area, like: Replace (STONE) with (CLAY)

~5 Schematic saving similar to world edit: When selecting a region in a world/project, you should be able to save them as a schematic on your computer, like a cool build or something, then paste the schematic in a different world/project.

~5 This I think is the most important feature that's in world edit that's missing from editor mode, Percantage replacing, could instantly make block pallets without you having to replace blocks with similar looking blocks to make builds actually look good, example: //replace stone 70%stone,20%andesite,5%gravel,5%dead_brain_coral_block

**Comment by minecraftchloe** [comment]
@AliveJy AWESOME feedback!! Thank you so much! I think with the v0.4 API Update, most of your feedback can be addressed by making an extension! 

That said, what you've listed also overlaps with what we'd like to implement into the core experience of the Editor, too! ✨
